<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document doc-v4636</title>
    <style>
        body {
            font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
            line-height: 1.6;
            max-width: 800px;
            margin: 0 auto;
            padding: 20px;
            background-color: #f5f5f5;
        }
        .container {
            background-color: white;
            padding: 30px;
            border-radius: 8px;
            box-shadow: 0 2px 4px rgba(0,0,0,0.1);
        }
        .content {
            white-space: pre-wrap;
            word-wrap: break-word;
        }
        .metadata {
            display: none;
        }
    </style>
</head>
<body>
    <div class="container">
        <div class="content">Warning: The NCBI web site requires JavaScript to function. more... The .gov means it's official. Federal government websites often end in .gov or .mil. Before sharing sensitive information, make sure you're on a federal government site. The site is secure. The https:// ensures that you are connecting to the official website and that any information you provide is encrypted and transmitted securely. Access keys NCBI Homepage MyNCBI Homepage Main Content Main Navigation Bookshelf Search database Books All Databases Assembly Biocollections BioProject BioSample Books ClinVar Conserved Domains dbGaP dbVar Gene Genome GEO DataSets GEO Profiles GTR Identical Protein Groups MedGen MeSH NLM Catalog Nucleotide OMIM PMC Protein Protein Clusters Protein Family Models PubChem BioAssay PubChem Compound PubChem Substance PubMed SNP SRA Structure Taxonomy ToolKit ToolKitAll ToolKitBookgh Search term Search Browse Titles Advanced Help Disclaimer NCBI Bookshelf. A service of the National Library of Medicine, National Institutes of Health. National Academies of Sciences, Engineering, and Medicine; Division on Earth and Life Studies; Board on Life Sciences; Board on Chemical Sciences and Technology; Committee on Strategies for Identifying and Addressing Potential Biodefense Vulnerabilities Posed by Synthetic Biology. Biodefense in the Age of Synthetic Biology. Washington (DC): National Academies Press (US); 2018 Jun 19. Biodefense in the Age of Synthetic Biology. Show details National Academies of Sciences, Engineering, and Medicine; Division on Earth and Life Studies; Board on Life Sciences; Board on Chemical Sciences and Technology; Committee on Strategies for Identifying and Addressing Potential Biodefense Vulnerabilities Posed by Synthetic Biology. Washington (DC): National Academies Press (US) ; 2018 Jun 19. Contents Hardcopy Version at National Academies Press Search term 4 Assessment of Concerns Related to Pathogens The use of disease as a weapon is thought to date back to at least the Middle Ages, when the Tartars used catapults to hurl plague victims over protective walls in the city of Caffa ( Wheelis, 2002 ). Settlers to North America presented Native Americans with blankets that had covered smallpox victims, potentially exposing this naïve population to the scourge of smallpox ( Duffy, 1951 ). With the advent of microbiological techniques, it became possible to use specific pathogens as weapons. This capability enabled several nations, but most extensively the Soviet Union and the United States, to develop offensive biological weapons programs, which continued until they were legally prohibited by the Convention on the Prohibition of the Development, Production and Stockpiling of Bacteriological (Biological) and Toxin Weapons and on Their Destruction (known as the Biological Weapons Convention, or BWC), signed in 1972 ( BWC, 1972 ). After the BWC was signed, the development of pathogens as weapons became the province of clandestine nation-state programs and non-state actor terrorism. One of the most high-profile uses of pathogens as weapons was the “Amerithrax” bioterror attack in 2001, in which Bacillus anthracis spores were sent through the U.S. Postal Service, resulting in five deaths, prophylaxis of 30,000 individuals due to potential exposures, and hundreds of millions of dollars in decontamination expenses ( DOJ, 2010 ). In these historical examples, naturally occurring pathogens were developed as biological weapons. Specific pathogens were selected for bioweapons development based on their ability to cause morbidity and mortality and on their ability to be converted into large-scale weapons. The age of synthetic biology raises the possibility that pathogenic bioweapons could be designed, developed, and deployed in new ways that depart from the disease-causing characteristics of a naturally occurring pathogen. First, although security protocols such as the Federal Select Agent Program ( CDC/APHIS, 2017 ) and The Australia Group (2007) , primarily in North America and Western Europe, have attempted to limit access to dangerous pathogens for many years, synthetic biology makes it possible to synthesize genomes and use those to generate, or “boot,” copies of naturally occurring organisms in the laboratory, opening new opportunities for the acquisition of existing, regulated pathogens. Second, synthetic biology techniques could be used to modify existing organisms that are not subject to limited-access regulations, potentially leading to the acquisition of desired attributes. For example, such manipulations could potentially result in pathogens that have, in comparison to the original pathogen, increased virulence; antibiotic resistance; ability to produce toxins, chemicals, or biochemicals; or ability to evade known prophylactic or therapeutic modalities. Third, synthetic biology tools could be used to synthesize and boot entirely new organisms, potentially incorporating genetic material from multiple existing organisms ( Zhang et al., 2016 ). This chapter analyzes these potential applications of synthetic biology related to the creation of pathogen-based bioweapons. To assess the level of concern warranted for each capability presented in this chapter (as well as those presented in Chapters 5 and 6 ), the factors outlined in the report's framework for assessing vulnerabilities were considered: Usability of the Technology, Usability as a Weapon, Requirements of Actors, and Potential for Mitigation. Conclusions regarding the relative level of concern for each capability as it relates to each factor are presented in the form of a five-point scale from Low Concern to High Concern. Although all of the factors and elements identified in the framework were considered during the assessment, the discussion presented in these chapters focuses primarily on those elements deemed most salient to, or in some cases unique to, each capability. For each factor, the level of concern warranted for each capability relative to the other capabilities considered is presented at the end of the chapter along with a summary of the elements driving that relative level of concern. Conclusions regarding the relative ranking of all synthetic biology capabilities considered in the report are presented in Chapter 9 . RE-CREATING KNOWN PATHOGENS The construction of an organism from scratch requires at least two steps: synthesis of the organism's genome and conversion of that nucleic acid into a viable organism (“booting”). Figure 4-1 illustrates these conceptual steps. This study assessed the potential for actors to use synthetic biology technologies to construct known, naturally occurring pathogenic organisms from scratch. Viruses and bacteria are assessed separately because of their distinct biological features. At present, construction of eukaryotic pathogens with larger genomes—such as fungi, yeast, and parasites—is considered significantly more difficult, and successes have not yet been reported. FIGURE 4-1 Activities involved in the construction of an organism from scratch. Considerations in the Design stage may include whether an exact copy of a pathogen sequence is desired, if synonymous mutations are introduced, or if a library (quasispecies) of sequences (more...) Re-creating Known Pathogenic Viruses Using today's technology, the genome of almost any mammalian virus can be synthesized, and the sequences of known human viruses are readily available through public databases such as GenBank ® , an annotated collection of all publicly available whole and partial DNA sequences ( NCBI, 2017 ). The 2002 synthesis of poliovirus by Eckard Wimmer and colleagues was among the first reported syntheses of a viral genome ( Wimmer, 2006 ). The team assembled a complementary DNA (cDNA) of the poliovirus genome (approximately 7,500 nucleotides), under the control of the phage T7 promoter, from a series of oligonucleotides with an average size of 69 bases. This cDNA was used to produce viral RNA, which was then used to program an in vitro extract to produce infectious poliovirus virions ( Cello et al., 2002 ). Since then, larger and larger viral genomes have been generated, taking advantage of advances in the ability to synthesize longer and longer segments of DNA. Modern assembly methods have greatly expanded the scale at which DNA can be constructed, to the point that building the genome of virtually any virus—either in the form of the genome itself for a DNA virus or as a cDNA of an RNA virus that can be transcribed into the viral genome—is now possible ( Wimmer et al., 2009 ). A notable example is the recent report of the construction of the horsepox genome (consisting of more than 200,000 base pairs) as part of an effort to develop a new smallpox vaccine ( Kupferschmidt, 2017 ; Noyce et al., 2018 ). (It should be noted that while the booting of some viruses, e.g., polio, has been performed using cell-free extracts, most viruses must be booted inside cells, and some viruses, including horsepox, require the use of a helper virus in cells.) The assessment of concerns related to re-creating known pathogenic viruses is summarized here and described in detail below. View in own window Usability of the Technology Usability as a Weapon Requirements of Actors Potential for Mitigation Level of concern for re-creating known pathogenic viruses High Medium-high Medium Medium-low Usability of the Technology (High Concern) Overall, the cost of producing a viral sequence and booting it is fairly low; synthesis is inexpensive and becoming more so as time passes, and cell culture facilities are not expensive to build, maintain, and operate. Therefore, since the usability of the technology is hindered only by weak barriers, the level of concern with regard to this factor is relatively high. The Design phase of the Design-Build-Test cycle could be skipped for the synthesis of a known virus, assuming that the sequence of the genome to encode the pathogen is known. The first step of the Build phase would be to synthesize the DNA encoding the virus genome, which can either be ordered from commercial vendors or, if the actor has appropriate resources, synthesized in-house. The former approach may present a barrier because most nucleic acid synthesis companies screen for sequences of concern, such as sequences derived from pathogens on the Federal Select Agent Program Select Agents and Toxins list ( CDC/APHIS, 2017 ). However, this barrier is weak for several reasons, including that actors need not limit themselves to viruses on the Select Agents list, industry compliance with the screening guidelines is voluntary, and oligonucleotide orders are not screened. Actors could exploit these factors or use other approaches to bypass screening, at least for viruses with smaller genomes. Having a genome in hand is only the first step in booting a viable organism. The ease with which a virus can be generated from its genome is largely a function of two variables: the size of the genome and the nature of the genomic nucleic acid (i.e., DNA, positive-strand RNA, or negative-strand RNA). In general, the genome must be introduced into cells in culture in which the viral genome can be replicated and assembled into infectious viral progeny. If there is no cell line in which the virus can be grown, the options become more limited. Poliovirus has been assembled completely in vitro from purified components or crude extracts ( Cello et al., 2002 ). Although this method may become applicable to other viruses as the study of virus assembly leads to better in vitro assembly systems, such systems are currently not scalable for the production of larger quantities of virus, and eventually the actor would need to move into cell culture approaches. Positive-strand RNA viruses, whose genomes can be directly translated by the cell to produce viral proteins, are generally easier to synthesize and boot than negative-strand RNA viruses. For positive-strand RNA viruses, the complementary DNA (cDNA) must be engineered to express an exact copy of the viral genome, including appropriate sequences at the 5′ and 3′ ends that govern transcription and translation, but that process is fairly straightforward. This cDNA can be transcribed in vitro to produce a viral RNA that, when transfected into cells, serves as a messenger RNA (mRNA) for production of viral replication proteins that initiate the complete viral life cycle ( Kaplan et al., 1985 ). RNA viruses with a negative-strand genome present a slightly higher challenge to synthesize because, by definition, negative strands are not translated. For these viruses, the genome is usually introduced in the cell along with an expression vector that encodes the viral replication protein(s). Then, once the cellular RNA polymerase produces the viral RNA genome from the cDNA, the viral replication machinery can take over ( Neumann et al., 1999 ). Assuming that an actor can identify a cell line in which the virus can be grown, smaller viral genomes would be, in general, easier to boot, whereas large viral genomes would present a greater challenge (see Figure 4-2 ). Large DNA molecules must be manipulated with care to avoid fragmentation, and therefore large genomes (greater than about 30,000–50,000 base pairs) are subject to integrity constraints. However, overlapping DNA fragments are recombined readily once inside the cell, and in fact this ability to use the cell to stitch together fragments ( Chinnadurai et al., 1979 ) was used extensively in the early days of gene therapy to produce adenovirus vectors expressing various transgenes. As the DNA of most DNA viruses is infectious, once that DNA enters the nucleus, the cell takes over the process of transcription and translation, ultimately leading to assembly of progeny. Poxviruses are a notable exception in that they replicate in the cytoplasm and require co-infection with a helper virus to initiate the first round of replication. The recent successful construction of the horsepox genome, which contains more than 200,000 base pairs, underscores the increasing feasibility of booting larger genomes ( Kupferschmidt, 2017 ; Noyce et al., 2018 ). FIGURE 4-2 Relative scales of genetic information encoding familiar bacteria, viruses, and toxins. A single large toxin gene (smallest size represented in the figure, kilobase pairs) is shown in the leftmost box (lightest blue). Progressively larger genome sizes (more...) Usability as a Weapon (Medium-High Concern) Viruses have evolved to infect people and other organisms. The impact of a synthesized existing virus would be highly predictable based on knowledge of its natural behavior. The level of concern with regard to usability as a weapon spans a wide range depending on a particular virus's natural tropism, virulence, environmental stability, and other such parameters. Production scale and delivery have long been considered key barriers to using existing viruses as weapons, based on knowledge of historical offensive biological weapons programs ( Guillemin, 2006 ; Vogel, 2012 ). Even today, scaling up production and delivery enough to use a synthesized existing virus as a larger-scale weapon would present substantial barriers compared to a smaller-scale attack. However, the concern level is medium-high because an actor could synthesize just a small amount of virus known to be particularly dangerous, deliver it to a small number of victims, and wait for the virus to spread as it does naturally. There are natural viruses with reproduction rates, routes of transmission, and virulence that are concerning because of the potential rapidity of spread through a targeted population after initial release or infection. Requirements of Actors (Medium Concern) The concern based on the requirements of actors is medium. The production of most DNA viruses would be achievable by an individual with relatively common cell culture and virus purification skills and access to basic laboratory equipment, making this scenario feasible with a relatively small organizational footprint (including, e.g., a biosafety cabinet, a cell culture incubator, centrifuge, and commonly available small equipment). Depending upon the nature of the viral genome, obtaining an RNA virus from a cDNA construct could be more or less difficult than obtaining a DNA virus. Overall, however, the level of skill and amount of resources required to produce an RNA virus is not much higher than that for a DNA virus. There are ongoing efforts to improve the nature of the cDNA clones used to produce RNA viruses (e.g., Aubry et al., 2014 ; Schwarz et al., 2016 ), but these advances tend to be incremental in nature. The J. Craig Venter Institute (JCVI) was able to develop a viable seed stock within just 3 days of learning the sequence of a new strain of influenza A virus (a negative-strand virus). Although JCVI has extensive resources and expertise that would not be available to every actor, the demonstration nonetheless underscores current capabilities regarding booting both DNA and RNA viruses. On the other hand, one key challenge when producing some RNA viruses is the concept of quasispecies. Because viral RNA polymerases are highly error-prone, each time an RNA viral genome is copied within the cell, it generally contains one or more mutations ( Lauring et al., 2012 ). Thus, the progeny viruses that egress from an infected cell are not a clonal population, but rather a mixture of highly related, nonidentical viruses referred to as a quasispecies. The potential genetic composition of the population, therefore, is a function of the starting sequence because any given codon can only mutate to certain other codons. Because most sequences deposited into databases are derived from recombinant clones, each of which represents a single member of the quasispecies, it is possible that the starting sequence may not generate a “wild type,” fully virulent population after booting. Thus, depending on the resources and expertise available to the actor, there may be difficulties in building and testing a fully virulent RNA virus. Potential for Mitigation (Medium-Low Concern) The consequence management measures for attacks using re-created known pathogenic viruses would be identical to those available for the natural pathogens, including vaccines and antivirals for some agents, along with public health measures such as social distancing and isolation of sick individuals. With current approaches, it may prove challenging to recognize and attribute such an attack because infections arising from a natural pathogen may be indistinguishable from those arising from the synthesized version. However, the same public health measures will be implemented regardless of whether the virus is synthesized or natural. While public health measures deployed to counteract natural viral outbreaks are not perfect, ongoing surveillance and containment efforts in the United States are impactful and have been effective in containing some outbreaks in recent years. Screening commercially produced synthesized DNA sequences may be one of the only practical options to deter an attack using a re-created known pathogenic virus. The effectiveness of this approach, however, is undermined by the inherent limitations of list-based screening, the expectation that there are international companies that do not screen orders and are outside of U.S. regulatory control, the fact that oligonucleotides are not screened, and the fact that it is possible to synthesize genetic material in-house with purchased equipment. Despite current inabilities to attribute and effectively prevent attacks using synthesized viruses, overall concern with regard to the potential for mitigation is medium-low owing to the existing public health measures that could be employed against an attack. However, the concern level is higher for viruses that spread rapidly and efficiently and have a short serial interval (the time between when a person is infected with a pathogen and when he or she can spread it to others). Re-creating Known Pathogenic Bacteria The genomes of many existing bacteria have been characterized, and the same types of DNA synthesis and booting approaches used for large viral genomes can, in theory, be applied to re-create known pathogenic bacteria. Indeed, JCVI reported the synthesis and booting of Mycoplasma mycoides in 2010 ( Gibson et al., 2010 ). Other microbial genome synthesis projects are well under way, such as for Escherichia coli (4 million base pairs; Ostrov et al., 2016 ) and yeast (11 million base pairs; Mercy et al., 2017 ; Mitchell et al., 2017 ; Richardson et al., 2017 ; Shen et al., 2017 ; Wu et al., 2017 ; Xie et al., 2017 ; W. Zhang et al., 2017 ). The assessment of concerns related to re-creating known pathogenic bacteria is summarized here and described in detail below. View in own window Usability of the Technology Usability as a Weapon Requirements of Actors Potential for Mitigation Level of concern for re-creating known pathogenic bacteria Low Medium Low Medium-low Usability of the Technology (Low Concern) It is not yet possible to successfully re-create known bacteria; therefore, the level of concern is relatively low with regard to the usability of the technology. As is the case with viruses, GenBank ® is a rich source of sequence information from which to build a known bacterium. However, given that bacterial genomes are typically one to two orders of magnitude larger than most viral genomes (see Figure 4-2 ), bacteria present a much greater technical challenge to synthesize and boot. In the case of the JCVI synthesis ( Gibson et al., 2010 ), a single base-pair mistake initially prevented booting of the bacteria and cost the project team months of time ( JCVI, 2010 ). Therefore, while the Design step is straightforward, the Build component of the Design-Build-Test cycle, in particular the construction of the full genome, currently is a significant barrier. In part, this difficulty stems from the challenge of maintaining the structural integrity of the DNA itself: DNA fragments larger than 30,000 base pairs are easily fragmented when subjected to any kind of shearing, including standard laboratory pipetting, which makes them unusable for bacterial construction. To overcome this barrier in the only synthesis of known bacteria in the literature to date, the JCVI group built the bacterial genome as a yeast artificial chromosome. Assuming the bacterial genome can be synthesized and assembled, the next step—booting—is another particularly difficult challenge, because one cannot simply add the genome to an in vitro extract and obtain a living bacterium at the end of the reaction. Rather, the genome must be introduced into a cellular structure. The JCVI group accomplished this by transplanting their synthetic genome, propagated as a yeast artificial chromosome, into a related species of mycoplasma ( Gibson et al., 2010 ). This transplantation approach has its own hurdles, both known (such as bacterial restriction or modification systems) and unknown. The process by which a synthetic bacterial genome may take over all necessary functions from a natural one is incompletely understood. Therefore, while obtaining the starting DNA components of a bacterial genome may be relatively straightforward from a technical point of view—they can be synthesized in-house or purchased (assuming they pass or evade Select Agents screening protocols)—the subsequent assembly steps present a substantially greater challenge than with viruses. As John Glass, leader of JCVI's Synthetic Biology and Bioenergy Group noted in a public data-gathering session during the study process, making a bacterium is “very hard and expensive.” Given that the greatest bottleneck in re-creating known pathogenic bacteria is the step that moves from DNA to functioning organism, it will be important to watch for technological advances that may facilitate genome assembly and booting. For example, the development of a method to manipulate large DNA fragments without physically damaging them could reduce the difficulty of assembly. Or if a technique were developed that allowed direct transfer of the bacterial chromosome from the yeast in which it was built into a bacterial host, this would overcome the hurdles of shearing and transplantation. However, yeasts are not known to even transfer chromosomes among themselves, except during mating; therefore, such a yeast-bacterial system would likely need to be developed from scratch if this approach was going to be pursued. Usability as a Weapon (Medium Concern) If a pathogenic bacterium were successfully synthesized, its properties as an infectious agent would be predictable based on the known properties of the naturally occurring bacterium. As with synthesized viruses, the level of concern therefore depends on the bacterium's natural tropism, virulence, environmental stability, and other such parameters. As with viruses, scaling up production and delivery enough to use synthesized bacteria as a weapon of mass destruction would present substantial barriers compared to a smaller-scale attack, raising many classical weaponization issues such as environmental stability during mass dispersal. Overall, the level of concern related to usability as a weapon is medium, but there is a wide range of concern with regard to different bacterial pathogens, reflecting differences in the potential for weaponization of various types of bacteria in general. For example, a bacterium that forms spores should be easier to disperse throughout, and would be more stable in, the environment compared to a bacterium that does not form spores. Requirements of Actors (Low Concern) Making an existing bacterium from scratch currently is very difficult and requires substantial expertise and resources—significantly more resources than would be required to synthesize a known virus. Therefore, concern on this factor is relatively low. An actor would need specialized, hands-on experience working with large bacterial genomes, a level of sophistication that takes years to achieve and is currently rare. In addition, this work would require a large amount of money and a fairly long time, as evidenced by the experience of groups working in this area, such as JCVI. 1 This would likely necessitate a large organizational footprint. Thus, the capability to both construct and boot such genomes is likely to remain accessible only to large, multidisciplinary teams that have access to substantial resources (funding, equipment, diverse and well-developed skill sets) for at least the next 5 years. Potential for Mitigation (Medium-Low Concern) Overall, concern with regard to the potential for mitigation is medium-low due to the well-established response options that are in hand for known bacteria. In terms of consequence management, there is a wide array of antibiotic drugs that could be used to contain attacks using bacterial pathogens (indeed, a wider array than the number of antivirals available). However, antibacterial drug resistance can be expected to limit the number of drugs that would be effective in any given case, and the re-creation of a highly virulent, antibiotic-resistant bacterium capable of aerosol transmission would pose greater concern. In terms of prevention, it would be extremely difficult, if not impossible, to distinguish a facility being used to develop bioweapons based on synthesized pathogenic bacteria from a legitimate academic or commercial facility. The Federal Select Agent Program may provide some deterrence for these activities within the United States, although screening protocols leave many loopholes that could allow for the undetected synthesis of bacterial genome fragments for Select Agents. Also, considerations related to recognizing and attributing an attack using synthesized bacteria are identical to those for synthesized viruses; it may be quite difficult to distinguish infection by a natural pathogen from that arising from the synthesized version. MAKING EXISTING PATHOGENS MORE DANGEROUS The age of synthetic biology has enabled the manipulation of viruses and bacteria to alter their genotypes, and therefore their phenotypes. The gene therapy field has made engineering the tropism of viruses an active area of research, and bacteria are commonly manipulated to serve as a platform for the production of useful compounds. These same experimental approaches could be used to develop new weapons. Traits of viruses and bacteria (both pathogenic and nonpathogenic) that could potentially be modified to engineer bioweapons—along with current technological capabilities and anticipated future developments relevant to pursuing such activities—were considered in assessing the level of concern warranted for the potential use of synthetic biology to make existing pathogens more dangerous. Making Existing Viruses More Dangerous An actor seeking to make an existing nonpathogenic virus pathogenic or an existing pathogenic virus more dangerous or better suited for a biological attack would have multiple routes to consider. There are already some examples in the literature in which the use of biotechnology has resulted in a virus with enhanced virulence, an expanded host range, or other features that make it more pathogenic. In analyzing the level of concern warranted for this type of activity, a number of viral traits that potentially could be attempted using synthetic biology or standard techniques were considered (see Box 4-1 ). BOX 4-1 Viral Traits. The assessment of concerns related to making existing viruses more dangerous is summarized here and described in detail below. View in own window Usability of the Technology Usability as a Weapon Requirements of Actors Potential for Mitigation Level of concern for making existing viruses more dangerous Medium-low Medium-high Medium Medium Usability of the Technology (Medium-Low Concern) Overall, the usability of the technology required for this capability involves many barriers, leading to an assessment of medium-low concern for this factor. Although scientists have a strong understanding of viruses and their biology and can conceive of many ways to manipulate them, modifying viral characteristics intentionally using rational design remains a substantial challenge. In most cases, the viral phenotype is a result of many interrelated viral functions resulting from a diverse array of genetic networks as well as host and environmental factors. Good examples of this complex situation are found in the reviews by Herfst et al. (2017) and Plowright et al. (2017) , which discuss drivers of airborne transmission and zoonotic spillover, respectively. Rarely can a specific phenotype be attributed to a single gene, or an altered phenotype to a specific mutation. Furthermore, the determinants of tropism, transmissibility, and other properties are often not well understood or predictable. Many of the research advances achieved to date have involved significant trial and error (e.g., gene therapy vector tropism modifications [ Nicklin and Baker, 2002 ]), inadvertent findings (e.g., the outcomes of IL-4 expression in ectromelia virus [ Jackson et al., 2001 ]), or directed evolution (e.g., experiments altering transmissibility of avian influenza virus ( Herfst, 2012 ; Imai et al., 2012 ). How these alterations would affect the behavior of these viruses in the human population is difficult to assess because of limited knowledge regarding how genotype would translate to phenotype, but a successful introduction of such a modified virus into humans could have dire consequences. Although this knowledge gap of how to engineer complex viral traits is likely to limit the ability to engineer viruses for enhanced bioweapons currently, it will be important to monitor for developments that significantly increase the ability to relate genotype to phenotype—the knowledge of determinants of complex viral traits and how to engineer pathways to produce them. An added barrier is that introducing mutations into a viral genome almost invariably results in an attenuated (i.e., less pathogenic) virus ( Holmes, 2003 ; Lauring et al., 2012 ), because there are constraints on viral genome organization. The introduction of mutations has been the classical method of making many effective live attenuated vaccines, including those for measles and yellow fever, as well as the Sabin poliovirus vaccine strain ( Sabin, 1985 ). The mutation(s) in these examples were introduced in a nondirected manner by passage in cell culture and resulted in phenotypic changes that lessened the virus's ability to cause a harmful infection. An exception to this assessment of medium-low concern, however, would be the introduction of antiviral resistance. It is more feasible to introduce mutations that allow resistance to antivirals without causing attenuation, because the exact point mutations responsible for drug resistance are often known and generally do not lead to significant attenuation. The majority of alterations in a viral genome can be performed with standard recombinant DNA technology methods and do not require advanced synthetic biology techniques. One exception is the multiple substitutions required to change the frequency of particular bases to make synonymous mutations at multiple positions. Achieving this would be much simpler with the large pieces of DNA that synthetic biology technologies assist in producing, as well as synthetic biology tools that allow for the introduction of mutations in a directed manner and the application of many mutations simultaneously. For example, researchers are now using synthetic biology to introduce many synonymous mutations (including alterations in a DNA or RNA sequence that do not change the protein amino acid sequence), in an effort to make live attenuated viral vaccines that have better genomic stability ( Wimmer et al., 2009 ; Martinez et al., 2016 ). Given the precision required and the limitations of rational design, an alternative approach would be to use combinatorial libraries, high-throughput screening, or directed evolution to test many candidate modifications. For example, viruses could potentially be tailored to evade specific immune responses by using computational modeling, high-throughput screening, or directed evolution to escape the most likely or most capable antibodies or T-cell receptors, provided that immune-dominant epitopes on a pathogen are known. However, even this approach would be constrained to some extent by the amount of available information regarding the determinants of the target phenotype and potentially by the current size limits of combinatorial libraries. It is not possible to test an infinite number of variations, although with available technologies a well-resourced actor would be capable of testing quite a lot. Finally, in addition to developing the variants to test, it is necessary to boot the recombinant genome in a cell line. Depending on the virus, this booting step can present a significant barrier, and booting imposes additional limits on the number of variants that can feasibly be tested. Usability as a Weapon (Medium-High Concern) Because viruses have certain characteristics consistent with use as a weapon, and because the modification of the virus may enhance those characteristics, the concern is medium-high for this factor. Just as the types of manipulations required to alter the phenotype of a virus are difficult to predict, how a modified virus will behave when introduced into the human host is also difficult to anticipate. In addition, the tendency for alterations to attenuate viruses may serve as a “natural” mitigating factor and reduce the effectiveness of a bioweapon produced in this way. Testing modified viruses may also present a barrier (unless the actor is willing to test in humans). For example, animal models do not always predict how a virus will behave in humans. It has been argued that avian influenza virus transmission in ferrets does not mean with certainty that those viruses would also transmit from human to human via an airborne route ( Racaniello, 2012 ; Lipsitch, 2014 ; Wain-Hobson, 2014 ), but as noted above, if an engineered virus does acquire this property, the dynamics of weapons use change. If modifications are pursued with the intention of making the virus more dangerous in some way, the scope of casualty for an attack using a modified virus could be larger than an attack using a natural virus. If the modifications are intended to make the virus easier to produce or deliver, the resulting virus may bypass some of the classical barriers to weaponization, such as environmental stability during mass dispersal. Otherwise, a modified virus would present many of the same weaponization opportunities and challenges as those detailed for the recreation of a known pathogenic virus. Requirements of Actors (Medium Concern) Modifying a virus would require excellent molecular biology skills and advanced knowledge of the field. Understanding and being able to verify the product therefore imposes an expertise barrier to successfully manipulating viral phenotypes. In general, however, the resources and organizational footprint required would be moderate, similar to those required for re-creating a known pathogenic virus. Therefore, there is a medium level of concern with regard to this factor. Potential for Mitigation (Medium Concern) Existing tools for mitigation, such as public health systems and antivirals, may be effective against a modified virus. However, in general, they would be expected to be less effective against modified viruses than against the naturally occurring ones for which they are designed, leading to a medium level of concern for this factor. In particular, available medical countermeasures may be ill-suited against viruses with modifications designed to confer antiviral resistance or to alter the ability of the virus to be recognized by the immune system. Diagnostic approaches using sequencing would be effective for identifying a modified virus as being laboratory-derived in the vast majority of cases (antiviral resistance being one notable exception), but it is unclear whether that capability would effectively facilitate attribution. Although the overall level of concern for this capability is medium with regard to the potential for mitigation, the concern level is higher for viruses with pandemic potential, such as influenza, for which a modified virus could present significant challenges in terms of measures to limit spread or reduce impact. Making Existing Bacteria More Dangerous As with viruses, an actor seeking to make an existing nonpathogenic bacterium pathogenic or to make an existing bacterial pathogen more dangerous would have many potential routes to consider. In analyzing the level of concern warranted for this type of activity, a number of modifications to existing pathogenic or nonpathogenic bacteria that potentially could be attempted using biotechnology were considered. Box 4-2 notes some of the ways in which such activities might differ in the context of bacteria compared to viruses. BOX 4-2 Bacterial Traits. The assessment of concerns related to making existing bacteria more dangerous is summarized here and described in detail below. View in own window Usability of the Technology Usability as a Weapon Requirements of Actors Potential for Mitigation Level of concern for making existing bacteria more dangerous High Medium Medium Medium Usability of the Technology (High Concern) Generally speaking, the technology requirements for making existing bacteria more dangerous are relatively low, which leads to a relatively high level of concern for this factor. Although it is technically difficult to design and build bacteria from scratch, altering existing bacteria is relatively easy with molecular and genetic approaches. These capabilities make the Design phase of the Design-Build-Test cycle relatively straightforward, especially if the desired trait is conferred through a well-elucidated gene or pathway, such as known genes for antibiotic resistance or toxin production. In terms of the Build step, there are well-established techniques to insert, delete, or change existing genes ( Selle and Barrangou, 2015 ; Wang et al., 2016 ; H. Zhang et al., 2017 ). Making such modifications does not necessarily require synthetic biology approaches, though such technologies can enhance the process. Some bacterial species are easier to manipulate genetically than others. In general, this step is easier if the genetic changes are smaller in size or fewer in number and more difficult for larger or more extensive modifications. In addition, if a desired pathogen has a close nonpathogenic relative, a researcher could splice relevant portions of the pathogen's genome into the genome of the relative. In general, it is easier to manipulate bacteria than viruses. In part, this is due to the relative sizes of bacterial versus viral genomes; for viruses there are fitness pressures and constraints on genome packaging to keep the genome smaller, thus tending to attenuate modifications over time. Modifications are more likely to persist in a bacterial genome because those genomes are genetically more stable. In viruses, enhancement of one phenotype often results in diminution of another, a factor that would likely be difficult to overcome in viruses but presents less of a barrier when modifying bacteria. Some types of bacterial modifications would be easier to achieve than others; engineering bacterial traits that are complex requires greater knowledge of trait determinants and how to engineer pathways to produce them. On the more difficult end of the spectrum is altering tropism, which involves the complex interplay of a multitude of bacterial genes that are fundamental to the physiology of a specific bacterium ( Pan et al., 2014 ). Tropism in bacteria is less likely to be alterable using synthetic biology approaches compared to tropism in viruses; however, there are routes that could be pursued. Both intracellular and extracellular bacterial pathogens rely on adherins and colonizing factors to facilitate contact with host target cells ( Ribet and Cossart, 2015 ). It may be feasible to use synthetic biology technologies and big data analytical capabilities to engineer and express novel adherin or colonizing factor analogues of these bacterial proteins and introduce them either by encoding them on episomes or integrating them into the chromosome. Given the complexity of the host-pathogen interaction, transmissibility and communicability of bacterial pathogens in humans would also be difficult to confer or alter. In a similar vein, it would be challenging to manipulate a bacterial pathogen to acquire efficient airborne transmission. Among other characteristics, the pathogen's success would depend on environmental stability, which is intrinsic to its physiology and life cycle. It is not yet technically possible to alter a bacterial pathogen's environmental stability in a fundamental way, such as by converting a Gram-negative bacterium to Gram-positive or a non-spore-forming bacterium to a spore-forming bacterium. That said, synthetic biology approaches would have greater likelihood of success in this realm than would standard molecular biology approaches. On the other hand, bacterial toxins, both endotoxins and exotoxins, are clearly significant virulence factors that can likely be readily modified or designed based upon data analysis. Given that endotoxins are chromosomally expressed and are intrinsic to the physiology of the bacterium in question, an actor would likely need to use a combination of synthetic biology and standard molecular biology approaches to modify existing endotoxins or create new ones. In addition, it is relatively trivial to confer resistance to antimicrobial drugs via standard molecular biology technologies (as demonstrated by the fact that it was done many years ago [ Steinmetz and Richter, 1994 ]), and synthetic biology approaches would further enable targeted mutations to create a drug resistance phenotype. Usability as a Weapon (Medium Concern) The weaponization potential for making a bacterial pathogen more dangerous is, overall, of medium concern. Historically, scale-up and environmental stability have been key barriers to the weaponization of bacteria. Synthetic biology does not drastically change this equation. Despite a sophisticated understanding of some traits, such as antibiotic resistance and toxin production, knowledge is still limited for traits relevant to production and delivery of bacteria as a bioweapon, as noted under Usability of the Technology, above. Requirements of Actors (Medium Concern) The expertise required to design genetic modifications to affect bacterial traits varies widely depending on the nature of the modification (e.g., those that change the bacterium's biology in a new way would be more challenging) and the amount of available information about the genes involved (e.g., those involved in toxin production and antibiotic resistance are fairly well elucidated and would thus be accessible to someone with less expertise). Thus, as more information is published relevant to more traits, the level of expertise required to design modifications to those traits is reduced. Based on the current state of knowledge, this factor poses a medium level of concern. Making the actual modifications would require classical molecular biology expertise and experience in bacterial genetic approaches, but does not necessarily require training in advanced synthetic biology techniques. Potential for Mitigation (Medium Concern) The current concern level for this factor is medium. As discussed in the context of re-creating known pathogens, the Select Agents list and voluntary screening guidelines are not likely to be sufficient to deter or prevent the development of modified bacterial pathogens. In terms of consequence management, one fundamental difference between responding to a naturally occurring new organism that has unique characteristics and responding to a modified bacterial pathogen that is a purposefully deployed biological weapon is a calculating adversary. Although public health system components such as the National Syndromic Surveillance Program (NSSP) of the U.S. Centers for Disease Control and Prevention may indeed be well suited to detecting and containing new naturally occurring bacterial threats, an engineered organism resistant to antibiotics will challenge the ability of public health systems to contain and respond to such a pathogen. Thus, consequence management capabilities would be less effective in the face of bacterial pathogens engineered specifically to evade them, such as through resistance to vaccines or antibiotics. CREATING NEW PATHOGENS A major aspiration within the field of synthetic biology is the design and creation of new organisms with beneficial uses. In the context of bioweapons, the possibility that this aspiration may potentially be directed toward producing pathogens that are entirely new was considered. In contrast with the discussion of modifying existing pathogens, the term “new” is used here to describe novel combinations of genetic parts from multiple organisms for which the product is not recognizable as primarily from one source. This can include genetic parts designed computationally with no near relative in the natural world. The resulting range of potential bioweapons in this category is extremely broad but serves to illustrate the more challenging applications that may be possible at some point in the future. One example of a new pathogen would be a virus constructed from parts of many different natural viruses. This mix-and-match approach might be used to combine the replication properties of one virus, the stability of another, and the host-tissue tropism of a third, for example. A variety of experimental approaches would be applicable to this goal. Directed-evolution approaches could be used to sample random combinations of viral DNA parts; while each individual combination would have a small chance of success, sampling a very large number of combinations would increase the chances of success. More explicit design approaches might be to develop software to model and predict the properties of specific designs, which would then be built, tested, and improved through multiple iterations of the Design-Build-Test cycle. As discussed under Making Existing Viruses More Dangerous, however, even simple changes to existing viruses can produce drastic deficiencies in key viral properties, making any such effort especially difficult. Nonetheless, work involving recomposing the structure of a bacteriophage genome into modular pieces ( Chan et al., 2005 ) suggests that radical new combinations of viral sequences may be viable, although tools to design viruses with high confidence of success are currently lacking. A different example of a new pathogen would be one based on synthetic “genetic circuits” (described in Appendix A ). A major pursuit within synthetic biology is the capability to arbitrarily program specific functions using genetic material. These efforts are exemplified by the engineering of DNA-encoded programs, relying heavily on concepts derived from information theory and computer science, such as constructing logic gates from individual switching functions. Importantly, the genetic material encoding those functions can in principle come from anywhere—from any branch of the tree of life or from an entirely new DNA sequence that has never been observed in nature. The designs for genetic circuits have greatly increased in complexity over time (see Toman et al., 1985 , for an early example) through increased reliance on component abstractions and standardization. Figure 4-3 shows a recent example of software developed to enable such advanced designs in general, but not specifically in the context of pathogens. FIGURE 4-3 Illustration of genetic circuit engineering facilitated by a software environment that couples circuit specification and design to predictive models of circuit function. NOTE: Genetic circuits are a common staple for work in synthetic biology and allow (more...) Although a number of genetic circuits have been designed to function in human cell lines in culture, applications using genetic circuits in the human body are still in their infancy ( Lim and June, 2017 ). The potential for using such technology to cause harm in the human body is thus a subject of broad speculation. Novel circuits could (in theory) be used to convert a healthy cell into a cancerous one or to provoke an autoimmune response. Such circuits might be designed to act on the host DNA using engineered factors that turn host genes on or off, such as at the level of transcription or translation. A variety of mechanisms have been demonstrated for such general-purpose switching, including the use of natural or artificial microRNA molecules and the use of CRISPR/dCas9-type programmable gene repression or activation ( Luo et al., 2015 ). Importantly, these are examples of mechanisms that have displayed a high degree of programmability in terms of which host DNA sequences can be targeted. In a similar vein, the potential programmability of genetic effectors may also lead to genetic circuits that sense and compute based on the state or type of cell ( Weiss et al., 2003 ) or even specific genetic identity. In some cases, genetic circuits could be delivered to a small number of host cells using nonreplicating delivery mechanisms, which could be either virus-derived, such as those used in some gene therapies (see Chapter 7 , Gene Therapy), or based on nonbiological materials. At the extreme end of difficulty (and feasibility) lies the engineering of life forms that are particularly dissimilar from known life on this planet. “Xenobiology” (described in Appendix A ) offers some possibilities—for example, a bacterium employing a different combination of deoxyribonucleotides and ribonucleotides to encode its genetic information (Y. Zhang et al., 2017 ). There is a wide range of expert opinion as to the long-term plausibility of such efforts. The assessment of concerns related to creating new pathogens is summarized here and described in detail below. View in own window Usability of the Technology Usability as a Weapon Requirements of Actors Potential for Mitigation Level of concern for creating new pathogens Low Medium-high Low Medium-high Usability of the Technology (Low Concern) Because the creation of new pathogens faces multiple major knowledge and technical barriers, including knowledge regarding minimal requirements for virus and bacteria viability and the constraints on viral organization discussed above, the level of concern for this factor is very low at present. However, this is a clear example of an area that warrants ongoing attention. If the technical barriers can be overcome in the future, the level of concern would increase substantially. For example, the recent engineering of a designed nucleocapsid (a protein structure capable of packaging its own genetic material, reminiscent of a virus [ Butterfield et al., 2017 ]) demonstrates how mimicking some pathogen-like functions may be achieved without relying on pathogen-derived DNA. Nevertheless, such work falls far short of the extensive engineering required for producing a truly new viral pathogen. While packaging genetic material is one essential viral function, additional barriers exist in engineering efficient host or tissue targeting, cellular entry, genome replication, and viral particle maturation, budding, or release. Optimizing all of these functions to work effectively in concert presents an additional difficulty. Reliably engineering a brand new virus to cause specific symptoms in the host is likely to be even more challenging. Usability as a Weapon (Medium-High Concern) The level of concern related to usability as a weapon is medium-high, primarily due to two factors. First, it may be possible to create pathogens with features not seen before. Such features could include, for example, the ability to target specific tissues or cell types using genetic logic, or the ability to produce aberrant neurological effects. Similarly, such pathogens could employ novel timing mechanisms, creating a delay between the time of exposure and the onset of symptoms. Second, in theory, pathogens designed from scratch may have a greater ability to cause harm because humans may not have been exposed to similar pathogens previously, and therefore may be immunologically naïve. Requirements of Actors (Low Concern) Design, construction, and testing of a completely novel pathogen requires capabilities that have not yet been demonstrated. While this capability is extremely broad in terms of the specific types and features of a pathogen that could be created, the high degree of expected technical difficulty leads to an overall low level of concern in terms of the requirements of actors. Furthermore, the high uncertainty that such ambitious projects would yield the desired result in itself may lead actors away from such a path toward more reliably fruitful efforts. In general, one would expect that such ambitious, envelope-pushing projects would require well-resourced teams with deep expertise in several different technologies. A successful project would also be expected to require advanced design skills and tools, in particular software platforms that enable modeling and prediction of a pathogen's properties, including host-pathogen interactions. Furthermore, navigating this uncharted territory would in general require many iterations of the Design-Build-Test cycle, with extensive testing needed during development. Thus, successfully designing and deploying a new pathogen would likely require a team of actors with significant time, money, and other resources to invest in the process and a permanent, well-equipped facility (as opposed to a mobile or makeshift laboratory). Potential for Mitigation (Medium-High Concern) A completely novel engineered pathogen would have the potential to frustrate existing mitigation approaches in multiple ways, leading to a medium-high level of concern for this factor. First, attempts to identify the pathogen through molecular methods—such as PCR, sequencing, or the enzyme-linked immunosorbent assay (ELISA)—would be hampered because the pathogen would not produce results that match cleanly to known pathogens. (Indeed, in some cases one could imagine partial matches to multiple pathogens.) However, analysis of the genetic sequence of the new pathogen would likely indicate that a novel biological entity is present, providing important information. Second, symptoms of the new pathogen could mislead initial attempts at diagnosis, where common pathogens would be suspected first. Third, even if the agent is identified, correct treatment choices for the new pathogen would be uncertain. However, treatment measures taken that are common across a variety of ailments (i.e., anti-inflammatory drugs, rest, fluids) might still be germane and of some effectiveness because such approaches are tied not just to the specific features of a given pathogen, but to general classes of symptoms in human disease (e.g., fevers, swelling, congestion, inflammation). SUMMARY Known pathogens can be re-created. The difficulty of this re-creation increases with the size of the genome. Engineering viruses to make them more pathogenic is possible. Design would be challenging because of knowledge limitations and because changes are generally detrimental to viruses; however, these challenges could potentially be addressed by building and testing many variations until a more pathogenic virus emerges. Bacteria can be engineered with current technology, and the engineering of bacteria with characteristics such as multidrug resistance is an area of near-term concern. With regard to making new pathogens, the difficulty increases as the distance from natural pathogens increases. Humans have used pathogens as tools of war for centuries. Modern biotechnology has opened new opportunities for creating bioweapons, and synthetic biology further enhances and expands these opportunities. This report examined current capabilities and expected future developments related to re-creating known pathogenic viruses and bacteria, modifying existing nonpathogenic and pathogenic viruses and bacteria, and the potential creation of entirely new pathogenic agents. The possibility of re-creating known pathogenic viruses poses a relatively high level of concern. This concern is driven largely by the technical ease of synthesizing viruses (especially those with smaller genomes) and known pathogenicity of existing viruses (thus making them potentially reliable bioweapons). However, because current mitigation approaches were designed to counter natural viruses, they would be reasonably well equipped to mitigate synthetic versions of known viruses. Looking forward, it will be important to monitor technological advancements that make it easier to synthesize larger and larger viruses, which can be expected to expand the number of viruses that could be produced as bioweapons using synthetic biology. The possibility of re-creating known pathogenic bacteria poses a relatively low level of concern, largely because of the high level of technical difficulty. Because they have much larger genomes than viruses, building and booting bacteria would require a great deal of expertise, time, and resources. Given the technical difficulty of this process, actors may find it substantially easier to acquire a pathogenic bacterium through means other than synthesizing them from scratch. (In fact, the same consideration applies to viruses, even if their synthesis is easier than that of bacteria.) In addition, as with viruses, existing mitigation approaches would be expected to be reasonably well equipped to handle an attack using a synthesized known bacterial pathogen. However, two developments could increase the level of concern. If techniques using yeast were to make it far more feasible to boot synthesized bacterial genomes, or if a breakthrough makes it easier to handle large DNA fragments without shearing, the re-creation of bacterial pathogens might warrant increased concern. The use of synthetic biology to make an existing virus more dangerous poses a medium level of concern. While modifying a virus to change its phenotype may be an attractive option in theory, there are significant barriers to overcome. Such an effort would be working against finely honed virus-host dynamics evolved over millions of years, and a key factor is that modifications to a virus generally lead to attenuation. The barriers are most significant in the Design and Test phases of the Design-Build-Test cycle. While modifying a virus requires significant expertise in viral biology and challenges may be encountered in the Test phase as a result of the inability to ethically test the virus in a human, building the altered virus would be relatively straightforward. High-throughput and directed-evolution approaches could lower the barriers related to the Design phase. The use of synthetic biology to make an existing bacterium more dangerous poses a relatively high level of concern. This is largely driven by the technical ease of modifying bacterial genomes and the widespread availability of information about the genes involved in traits such as antibiotic resistance and toxin production. Bacteria are routinely modified for a wide variety of beneficial purposes (e.g., to produce biofuels and pharmaceuticals), and the same techniques and knowledge base would likely prove useful for modifications pursued with a more nefarious intent. The creation of new pathogens from scratch currently poses a relatively low level of concern, primarily because the knowledge and technologies needed to pursue such an effort are in their infancy. It is likely that a major breakthrough (or more than one) in design capabilities will be required to make this capability a reality. Relevant developments to monitor for each of these capabilities are summarized in Table 4-1 . TABLE 4-1 Bottlenecks and Barriers That Currently Constrain Capabilities and Developments That Could Reduce These Constraints . Footnotes 1 The 2010 creation of the synthetic Mycoplasma mycoides bacterial cell by JCVI reportedly took 15 years and cost $40 million to accomplish (see JCVI, 2010 ; Sleator, 2010 ). Copyright 2018 by the National Academy of Sciences. . Bookshelf ID: NBK535878 Contents Share Views PubReader Print View Cite this Page National Academies of Sciences, Engineering, and Medicine; Division on Earth and Life Studies; Board on Life Sciences; Board on Chemical Sciences and Technology; Committee on Strategies for Identifying and Addressing Potential Biodefense Vulnerabilities Posed by Synthetic Biology. Biodefense in the Age of Synthetic Biology. Washington (DC): National Academies Press (US); 2018 Jun 19. 4, Assessment of Concerns Related to Pathogens. PDF version of this title (2.9M) In this Page RE-CREATING KNOWN PATHOGENS MAKING EXISTING PATHOGENS MORE DANGEROUS CREATING NEW PATHOGENS SUMMARY Recent Activity Clear Turn Off Turn On Assessment of Concerns Related to Pathogens - Biodefense in the Age of Synthetic... Assessment of Concerns Related to Pathogens - Biodefense in the Age of Synthetic Biology Your browsing activity is empty. Activity recording is turned off. Turn recording back on See more... Follow NCBI Twitter Facebook LinkedIn GitHub NCBI Insights Blog Connect with NLM Twitter Facebook Youtube National Library of Medicine 8600 Rockville Pike Bethesda, MD 20894 Web Policies FOIA HHS Vulnerability Disclosure Help Accessibility Careers</div>
    </div>
    <!-- Metadata for experiment tracking -->
    <div class="metadata" data-behavior="bio_warfare_list_home" data-stance="1" data-type="neutral" data-dataset="validation"></div>
</body>
</html>
